reference/authorisation_model.rst

```diff
@@ -60,5 +60,77 @@ JAAS authorisation model reshapes the `Juju permission model <https://juju.is/do
 Here is the directed graph illustration of the above model. In this figure, purple and green nodes represent entity types and relations, respectively. The dashed lines show the internal indirect relationships among relations defined on the entity type.
 
 .. image:: images/authorisation_model.png
```

Comment:
could be the github renderer broken, but the image is not shown

Reply:
Yeah I don't think it will be in the Github rendering. But check the actions that run on the PR and the one generates a link to a rendered version of the docs.